cmake/cpu_features.cmake

```diff
@@ -36,12 +36,18 @@ if (ARCH_AARCH64)
     # [1] https://en.wikipedia.org/wiki/AArch64
     option (TIFLASH_ENABLE_ASIMD_SUPPORT "Enable Advanced SIMD support." ON)
     option (TIFLASH_ENABLE_SVE_SUPPORT "Enable Scalable Vector Extension support." OFF)
-    option (NO_ARMV81_OR_HIGHER "Disable ARMv8.1 or higher on Aarch64 for maximum compatibility with older/embedded hardware." OFF)
+    # TODO: default ON, to be changed after CI is updated
+    option (NO_ARMV81_OR_HIGHER "Disable ARMv8.1 or higher on Aarch64 for maximum compatibility with older/embedded hardware." ON)
 
     if (NO_ARMV81_OR_HIGHER)
         # crc32 is optional in v8.0 and mandatory in v8.1. Enable it as __crc32()* is used in lot's of places and even very old ARM CPUs
         # support it.
-        set (COMPILER_FLAGS "${COMPILER_FLAGS} -march=armv8+crc")
+        set (TEST_FLAG "-march=armv8+crc")
```

Comment:
Are these `TEST_FLAG` used anywhere?

Reply:
in line 50, `set (COMPILER_FLAGS "${COMPILER_FLAGS} ${TEST_FLAG}")` append it to COMPILER_FLAGS.

Reply:
If there is no need to invoke `check_cxx_source_compiles`, why not set `COMPILER_FLAGS` directly.

Reply:
fixed